jaxkern/stationary/rbf.py

```diff
@@ -64,8 +64,10 @@ def __call__(
         return K.squeeze()
 
     def init_params(self, key: KeyArray) -> Dict:
+        eps = 1e-3
```

Comment:
Is there a reason for `1e-3`. I’d personally like the jitter to be as small as possible.